core/src/main/scala/chisel3/VerificationStatement.scala

```diff
@@ -127,7 +164,16 @@ object assume {
   )(
     implicit sourceInfo: SourceInfo,
     compileOptions:      CompileOptions
-  ): Assume = macro _applyMacroWithMessage
+  ): Assume = macro _applyMacroWithStringMessage
+
+  def apply(
```

Comment:
Needs ScalaDoc, see the format String apply above for an example.

Reply:
Added - used the one present already as a template and made slight modifications! Hope that's fine.